### PR TITLE
grid component completed

### PR DESCRIPTION
**Pull request by yogesh-**
Added the following items in grid :
- grid 2 column layout
- grid 3 column layout

Browse the preview: https://deploy-preview-7--agniui.netlify.app/documentation

### CODE REVIEW

File: css/grid.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  padding: 2rem;
+}
+
+.grid-3-col {
+  grid-template-columns: 1fr 1fr 1fr;
+}
+
+.grid-item {
+  height: 10rem;
+  border: 0.2rem dotted #333333;
```
Comment (syed-shaida-hussain) [inline css/grid.css:15]: Try using variable for color . Rest looks good.

**Review by syed-shaida-hussain — CHANGES_REQUESTED** [review]

File: documentation.html [hunk 2]
```diff
@@ -1149,7 +1155,80 @@ <h2 class="h2-style">Center Text</h2>
...
+                            </iframe>
+                        </div>
+
 
+                    </div>
+
+
+                    <!-- 3 column grid -->
+
+                    <div class="card1">
```
Comment (akash-1318) [inline documentation.html:1201]: Don't add number in class name like this card1

File: css/grid.css (hunk 1 above)
Comment (arjunbharti) [reply css/grid.css:15]: Use variable name for colors

Comment (yogesh-) [reply css/grid.css:15]: done.

File: documentation.html (hunk 2 above)
Comment (yogesh-) [reply documentation.html:1201]: okay.

**Pull request merged**